std/haxe/coro/BaseContinuation.hx

```diff
@@ -67,7 +67,7 @@ abstract class BaseContinuation<T> extends SuspensionResult<T> implements IConti
 				completion.resume(null, @:privateAccess Exception.thrown(e));
 			}
 			#end
-        });
+        }, 0);
```

Comment:
Syntax like this makes me think that the signature should be `(time, func)` instead.

Reply:
Yes, I think that would look better. I just renamed the old `scheduleIn` which is why it's that way.